sematic/cli/cli.py

```diff
@@ -5,7 +5,7 @@
 import click
```

Comment:
This file change is here for illustrative and reproducibility purposes. It will be removed before merging. Its contents will be included in a dedicated PR that exposes the Build System to users.